public/js/angular/controllers.js

```diff
@@ -1,6 +1,6 @@
 angular
-  .module('webmakerApp')
-  .controller('navigationController', ['$scope', '$location', '$routeParams', '$rootScope', 'weblit', 'wmNav', 'CONFIG',
+  .module("webmakerApp")
```

Comment:
everything so far was replaced from " to ', why is this replaced the other way around?